api/src/services/listing-csv-export.service.ts

```diff
@@ -316,7 +318,18 @@ export class ListingCsvExporterService implements CsvExporterServiceInterface {
     return fieldValue;
   };
 
-  async getCsvHeaders(): Promise<CsvHeader[]> {
+  doAnyJurisdictionHaveFeatureFlagSet = (
+    jurisdictions: Jurisdiction[],
+    featureFlagName: string,
+  ) => {
+    return !!jurisdictions.find((juris) => {
```

Comment:
this can be `some` instead of `find` calls

Reply:
nice! fixed!